src/lsm/forest_fuzz.zig

```diff
@@ -273,41 +357,83 @@ const Environment = struct {
             const storage_size_used = env.storage.size_used();
             log.debug("storage.size_used = {}/{}", .{ storage_size_used, env.storage.size });
 
-            const model_size = model.count() * @sizeOf(Account);
+            const model_size = (model.log.readableLength() + model.checkpointed.count()) * @sizeOf(Account);
+            // FIXME: This isn't accurate anymore because the model can contain multiple copies of an account in the log
```

Comment:
That's unfortunate, but at least it's asymptotically accurate.

Reply:
I think we'd have to maintain a separate set data structure for this if we wanted to keep it, do you think it's worth the extra overhead?

Reply:
I think you can leave it as is. Maybe change the `=` in the log message to `~=` :)

Reply:
good idea